pgml-apps/discord-bot/docs/algorithm_selection.md

```diff
@@ -0,0 +1,106 @@
+# Algorithm Selection
```

Comment:
I don't think we should copy these files. My preference would be (without knowing feasibility):
1. direct path in repository
2. symlink to directory in repository
3. git submodule

Reply:
Direct path in repo is good, we don't need a submodule for the content found in the same git repo.